services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+
+) -> None:
+    user_to_update = get_user(user_id)
+
+    if username:
+        user_to_update.username = username
+    if password:
+        user_to_update.set_password(raw_password=password)
+    if email:
```

Comment:
I'd suggest to use `is not None` for the last three fields, since they can be blank.